src/platform/interpreter/constants.ts

```diff
@@ -299,54 +146,40 @@ export const BuiltInModules = [
     'pydoc',
     'pydoc_data',
     'queue',
-    'quit',
     'quopri',
     'random',
-    'range',
     're',
     'readline',
-    'repr',
     'reprlib',
     'resource',
-    'reversed',
     'rlcompleter',
-    'round',
     'runpy',
     'sched',
     'secrets',
     'select',
     'selectors',
-    'set',
-    'setattr',
     'shelve',
     'shlex',
     'shutil',
     'signal',
     'site',
-    'slice',
     'smtpd',
     'smtplib',
     'sndhdr',
     'socket',
     'socketserver',
-    'sorted',
```

Comment:
I would have thought these would all be back?

Reply:
No, those are not modules that can be imported, they are reserved names like functions/classes etc.
E.g. `eval` and `AssertionError` are classes, and we know these cannot be imported.
The error message focuses on modules that can be imported by python and override pacakges.
Even if you have such a file, python will not end up using them, unless we have it in a startup file that gets pulled into the python runtime that loads them and shadows the built ins

Reply:
If it's more likely they'd create false positives than actually catch something - better to leave them out, but maybe there was a way they could still interfere? I didn't quite understand the levels of uncertainty about the files